scripts/compare-file.js

```diff
@@ -0,0 +1,10 @@
+const fs = require("fs");
```

Comment:
wonder if we could use a command like `diff` (sets $? to 0 on no difference apparently) instead, but that's not always available on windows I suppose?

Just an idea, this is fine too.

Reply:
it is indeed not available :(